README.md

```diff
@@ -97,6 +97,44 @@ podman run --rm -p 8080:8080 \
   zot:latest
 ```
 
+# CLI
+
+The same zot binary can be used for interacting with any zot server instances.
+
+## Adding a zot server URL
+
+To add a zot server URL with an alias "aci-zot":
+
+```console
+$ zot config add aci-zot https://aci-zot.cisco.com:5000
+```
+
+List all configured URLs with their aliases:
+```console
+$ zot config -l
+aci-zot https://aci-zot.cisco.com:5000
+local   http://localhost:8080
+```
+
+## Fetching images
+An alias can be used to fetch all images hosted on a server:
```

Comment:
"can be" - make it a mandatory language.

"You can now fetch all images from a server by using it's alias specified in Step x (link)"